.circleci/config.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             . venv/bin/activate
             pip install --install-option="--no-cython-compile" cython
             pip install numpy
-            pip install nose mpmath argparse Pillow codecov matplotlib "Sphinx!=3.1.0" asv
+            pip install nose mpmath argparse Pillow codecov matplotlib "Sphinx!=3.1.0" asv pythran
```

Comment:
I went ahead and pushed in this missing install for CircleCI. Unrelated, but I wonder if that `nose` install can be removed at this point.

I'm less sure about the Appveyor failure from a very quick glance--the error output there is rather verbose and `pythran` is in the requirements file we have for appveyor.